src/functionalTest/resources/scenarios/RIA-815-appeal-saved-confirmation-start.json

```diff
@@ -15,8 +15,8 @@
   "expectation": {
     "status": 200,
     "confirmation": {
-      "header": "# Appeal saved\nYou still need to submit it",
-      "body": "#### Ready to submit?\n\n[Submit your appeal](/case/IA/Asylum/1234/trigger/submitAppeal) when you are ready.\n\n#### Not ready to submit yet?\nYou can return to the case to make changes."
+      "header": "# Your appeal details have been saved\n# You still need to submit it",
+      "body": "### Do this next\n\nIf you're ready to proceed [submit your appeal](/case/IA/Asylum/1234/trigger/submitAppeal).\n\n#### Not ready to submit yet?\nYou can return to the case details to make changes."
```

Comment:
Just to double-check this is all right. Are we sure we want to change the message for other ticket and scenarios?